src/Uno.UI/UI/Xaml/Controls/DatePicker/DatePickerFlyout.iOS.cs

```diff
@@ -31,9 +31,16 @@ private void InitializeContent()
 
 		private void DatePickerFlyout_Opening(object sender, EventArgs e)
 		{
+			// The date coerced by UIDatePicker doesn't propagate back to DatePickerSelector (#137137)
+			// unless the user selected an invalid date, which an ValueChanged will be raised after coercion.
+			// As a workaround, we clamp the date between the valid range.
```

Comment:
I think we can remove "As a workaround" (or replace with "To address this" or something similar), if this is the best we can do it's not really a workaround :)